apisix/plugins/prometheus/exporter.lua

```diff
@@ -530,16 +617,16 @@ end
 _M.get_api = get_api
 
 
-function _M.export_metrics(stream_only)
+function _M.export_metrics()
     if not prometheus then
-        core.response.exit(200, "{}")
+       return core.response.exit(200, "{}")
```

Comment:
JFI, this behavior seems to be inconsistent with what is in get_cached_metrics and we need to confirm which mode should be used. cc @membphis 

BTW, prometheus to nil will happen when the plugin is dynamically disabled.